src/main/java/core/basesyntax/service/impl/ReturnAction.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.service.ActionHandler;
+
+public class ReturnAction implements ActionHandler {
```

Comment:
The class 'ReturnAction' should not start with an empty line according to the checklist.